src/sphere.h

```diff
@@ -70,7 +70,7 @@ class Sphere : public Obstacle
      *  \param step, step vector where to move.
      *  \brief Returns true if it was any analytical collision to the infinite plane
      */
-    inline bool handleCollition(Walker& walker, Collision &colision, Eigen::Vector3d& step,double& a,double& b, double& c,double& discr,double& step_length);
+    inline bool handleCollition(Walker& walker, Collision &colision, Eigen::Vector3d& step,double& a,double& b, double& c,double& discr,double& step_length, double &D_in, double &D_ex);
```

Comment:
This new method, with diff_in n diff_out, shouldn't remove the previous one, it should have been that the method with no double &D_in, double &D_ex  just sets the same diffusivity passed as argument as before, so it's compatible with the older versions.